internal/consts/consts.go

```diff
@@ -4,183 +4,194 @@ const (
-	FieldScope                    = "scope"
-	FieldSubscriptionID           = "subscription_id"
-	FieldResourceGroupName        = "resource_group_name"
-	FieldVMName                   = "vm_name"
-	FieldVMSSName                 = "vmss_name"
-	FieldUsernameFormat           = "username_format"
-	FieldIntegrationKey           = "integration_key"
-	FieldAPIHostname              = "api_hostname"
-	FieldPushInfo                 = "push_info"
-	FieldUsePasscode              = "use_passcode"
-	FieldIssuer                   = "issuer"
-	FieldPeriod                   = "period"
-	FieldKeySize                  = "key_size"
-	FieldQRSize                   = "qr_size"
-	FieldAlgorithm                = "algorithm"
-	FieldDigits                   = "digits"
-	FieldSkew                     = "skew"
-	FieldMaxValidationAttempts    = "max_validation_attempts"
-	FieldOrgName                  = "org_name"
-	FieldAPIToken                 = "api_token"
-	FieldBaseURL                  = "base_url"
-	FieldPrimaryEmail             = "primary_email"
-	FieldSettingsFileBase64       = "settings_file_base64"
-	FieldUseSignature             = "use_signature"
-	FieldIdpURL                   = "idp_url"
-	FieldAdminURL                 = "admin_url"
-	FieldAuthenticatorURL         = "authenticator_url"
-	FieldOrgAlias                 = "org_alias"
-	FieldType                     = "type"
-	FieldMethodID                 = "method_id"
-	FieldMFAMethodIDs             = "mfa_method_ids"
-	FieldAuthMethodAccessors      = "auth_method_accessors"
-	FieldAuthMethodTypes          = "auth_method_types"
-	FieldIdentityGroupIDs         = "identity_group_ids"
-	FieldIdentityEntityIDs        = "identity_entity_ids"
-	FieldWrappingAccessor         = "wrapping_accessor"
-	FieldRoleName                 = "role_name"
-	FieldPolicies                 = "policies"
-	FieldNoParent                 = "no_parent"
-	FieldNoDefaultPolicy          = "no_default_policy"
-	FieldRenewable                = "renewable"
-	FieldExplicitMaxTTL           = "explicit_max_ttl"
-	FieldWrappingTTL              = "wrapping_ttl"
-	FieldDisplayName              = "display_name"
-	FieldNumUses                  = "num_uses"
-	FieldRenewMinLease            = "renew_min_lease"
-	FieldRenewIncrement           = "renew_increment"
-	FieldLeaseStarted             = "lease_started"
-	FieldClientToken              = "client_token"
-	FieldWrappedToken             = "wrapped_token"
-	FieldOrphan                   = "orphan"
-	FieldVaultVersionOverride     = "vault_version_override"
-	FieldSkipGetVaultVersion      = "skip_get_vault_version"
-	FieldMemberEntityIDs          = "member_entity_ids"
-	FieldMemberGroupIDs           = "member_group_ids"
-	FieldExclusive                = "exclusive"
-	FieldGroupID                  = "group_id"
-	FieldGroupName                = "group_name"
-	FieldExternal                 = "external"
-	FieldInternal                 = "internal"
+	FieldAPIHostname                = "api_hostname"
```

Comment:
These have been sorted. I thought it might be ok do so since I was adding fields that were changing the indent level for all entries. However, I am beginning to doubt the value in doing this as it could make rebasing on top of this quite annoying.

Reply:
Hmm that is definitely a valid point! I'm now wondering if it's better to leave these unsorted and as they were for now, and we can decide on the benefits of sorting once we have finished updating all resources/datasources to use field constants — something we are tracking for long term sustainability of this project which will add to this list a lot. Might be worth adding all the constants and then sorting them in one go. What do you think?